### PR TITLE
Defer Dust access to IDP

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

If a user has been revoked, they cannot access the app unless they receive a valid invitation from an admin. This is the standard procedure for most scenarios. However, there is an exception for Enterprise connections (SSO) where Dust's permissions are superseded by the Identity Provider (IDP). Therefore, access to Dust should be managed through the IDP instead of directly through Dust.

### CODE REVIEW

**Review by philipperolet — COMMENTED** [review]
Are we sure about that?
Wouldn't it mean that if, as I admin, I revoke a user from dust via the Members page of dust, and not from elsewhere, said member will still be able to reaccess without admin approval?

**Comment by philipperolet** [comment]
As discussed IRL, great if we can prevent SSO-enabled workspaces from revoking from the dust UI

**Comment by flvndvd** [comment]
> As discussed IRL, great if we can prevent SSO-enabled workspaces from revoking from the dust UI

Done.

image

**Comment by flvndvd** [comment]
@philipperolet merging to unblock customers but happy to address any other feedback. PMRR.

**Review by JulesBelveze — APPROVED** [review]

File: front/components/members/InvitationModal.tsx
Comment (philipperolet) [inline front/components/members/InvitationModal.tsx:345]: so we had it after all? :+1: 

Would add a tooltip here "As SSO is enforced on your workspace, member revocation can only be done through..."

**Review by philipperolet — APPROVED** [review]
LGTM :+1: 
Would add a tooltip but can be done after

**Pull request merged**
